src/pages/Editor.vue

```diff
@@ -1968,7 +1959,9 @@ export default {
           this.hasUnpublishedChanges = false;
           this.videoDBId = plioDetails.videoDBId;
           this.plioDBId = plioDetails.plioDBId;
-          this.plioSettings = Utilities.setPlioSettings(this.loadedPlioDetails.config);
+          this.plioSettings = SettingsUtilities.setPlioSettings(
+            this.loadedPlioDetails.config
+          );
           this.constructSettingsMenu();
```

Comment:
can't you directly call `SettingsUtilities.prepareSettingsToRender` here instead of creating another new function to essentially do a function call?